config/crd/bases/ocs.openshift.io_ocsinitializations.yaml

```diff
@@ -1,169 +1,169 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
This is a problem, you're using an outdated version of controller-gen. Try deleting the binary we download inside the repo and running it again.